.pipelines/e2e-windows-template.yaml

```diff
@@ -0,0 +1,59 @@
+parameters:
```

Comment:
Can we move this file to `.pipelines/templates/` since it is a template?

Reply:
Sure. That's better.